src/core/storage/sframe_interface/unity_sframe.cpp

```diff
@@ -426,9 +426,9 @@ std::shared_ptr<unity_sarray_base> unity_sframe::select_column(const std::string
   size_t column_index = _column_index_iter - _column_names.begin();
   auto ret = select_column(column_index);
 
```

Comment:
code change for #2227; irrelevant to this PR; Feel free to ignore.

Reply:
Can you rebase your changes against master? It's better (for reviewers) if you put up independent PRs. Thanks!

Reply:
Nevermind, I see you actually have a dependency here.